.github/workflows/publish.yml

```diff
@@ -19,10 +25,7 @@ jobs:
       - name: Build package
         run: make build
```

Comment:
I recommend building the dists in a separate job that does not have access to the `id-token: write` privilege, as it's a potential security hole.

Reply:
Ahh, I see. Do the absolute minimum needed when you have the token. Thanks!